src/main/vfxutil.js

```diff
@@ -0,0 +1,59 @@
+export let transparency = true;
+
+export function getTransparency(){
+  return transparency;
+}
+export function toggleTransparency(){
+  transparency = !transparency;
+}
+
```

Comment:
should i treat this variable as its own entity and put these into a whole new file?